tests/accelerators/test_ipu.py

```diff
@@ -344,6 +344,38 @@ def test_autoreport(tmpdir):
     assert os.path.isfile(autoreport_path + "profile.pop")
 
 
+@RunIf(ipu=True)
+def test_manual_poptorch_dataloader(tmpdir):
+    model_options = poptorch.Options()
```

Comment:
Uhmm, so we assume that `poptorch` will always be installed? Should we also make sure that it's available in the decorator `@RunIf(ipu=True)`?

Reply:
`poptorch` will be installed if `ipu=True`

Reply:
Cool!